.test_mageck_mle/config/config.yml

```
@@ -0,0 +1,67 @@
+      non_essential_genes: none
+    extra_args: # Extra arguments for bagel2 subcommands
+      bf: ""
+      pr: ""
+  
+  mageck:
+    run: True # Perform mageck analysis
+    command: mle # test or mle
+    mle:
+      design_matrix: resources/matrix.txt # Design matrix for mageck mle
+    # It is recommended to disable crisprcleanr when using non-genome-wide sgRNA libraries
+    apply_crisprcleanr: False 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)